Libraries/Modal/Modal.js

```diff
@@ -55,9 +60,18 @@ class Modal extends React.Component {
       backgroundColor: this.props.transparent ? 'transparent' : 'white',
     };
 
+    let animationType = this.props.animationType;
+    if (!animationType) {
+      // manually setting default prop here to keep support for the deprecated 'animated' prop
+      animationType = 'none';
+      if (this.props.animated) {
+        animationType = this.props.animated ? 'slide' : 'none';
```

Comment:
nit: you've also checked the truthiness of `this.props.animated`, you can immediately set it to slide here.